0.6-dev/index.bs

```diff
@@ -0,0 +1,1787 @@
+            │
+            └── 0             # Multiscale, labeled image. The name is unimportant but is registered in the "labels" group above.
+                ├── zarr.json # Zarr Group which is both a multiscaled image as well as a labeled image.
+                │             # Metadata of the related image and as well as display information under the "image-label" key.
+                │
+                ├── 0         # Each multiscale level is stored as a separate Zarr array, as above, but only integer values
+                └── ...       # are supported.
+                
+</pre>
+
+
+
+High-content screening {#hcs-layout}
+------------------------------------
+
+The following specification defines the hierarchy for a high-content screening
+dataset. Three groups MUST be defined above the images:
+
+-   the group above the images defines the well and MUST implement the
+    [well specification](#well-md). All images contained in a well are fields
+    of view of the same well
+-   the group above the well defines a row of wells
+-   the group above the well row defines an entire plate i.e. a two-dimensional
+    collection of wells organized in rows and columns. It MUST implement the
+    [plate specification](#plate-md)
+
+A well row group SHOULD NOT be present if there are no images in the well row.
+A well group SHOULD NOT be present if there are no images in the well.
+
+
+<pre>
+.
+│
+└── 5966.zarr                 # One OME-Zarr plate (id=5966)
+    ├── zarr.json             # Implements "plate" specification
+    ├── A                     # First row of the plate
+    │   ├── zarr.json
+    │   │
+    │   ├── 1                 # First column of row A
+    │   │   ├── zarr.json     # Implements "well" specification
+    │   │   │
+    │   │   ├── 0             # First field of view of well A1
+    │   │   │   │
+    │   │   │   ├── zarr.json # Implements "multiscales", "omero"
+    │   │   │   ├── 0         # Resolution levels          
+    │   │   │   ├── ...
+    │   │   │   └── labels    # Labels (optional)
+    │   │   └── ...           # Other fields of view
+    │   └── ...               # Other columns
+    └── ...                   # Other rows
+</pre>
+
+OME-Zarr Metadata {#metadata}
+=============================
+
+The "OME-Zarr Metadata" contains metadata keys as specified below 
+for discovering certain types of data, especially images.
+
+The OME-Zarr Metadata is stored in the various `zarr.json` files throughout the above array
+hierarchy. In this file, the metadata is stored under the namespaced key 
+`ome` in `attributes`.
+The version of the OME-Zarr Metadata is denoted as a string in the `version` attribute within the `ome` namespace.
+
+The OME-Zarr Metadata version MUST be consistent within a hierarchy.
+
+```json
+{
+  ...
+  "attributes": {
+    "ome": {
+      "version": "0.5",
+      ...
+    }
+  }
+}
+```
+
+"axes" metadata {#axes-md}
```

Comment:
The way this reads now is confusing -- `"axes"` metadata is defined, but it's not clear where that field is actually used.

I was under the impression that this PR was moving all axis definitions to inside the `coordinateSystems` field. Is that true? If so, this section should probably be made a subsection of the section describing the `coordinateSystems`.